### PR TITLE
hardening/454_link_qsg_in_readme

### PR DESCRIPTION
**Pull request by frbattid**
* Fixes issue #454 
* No unit tests nor (unofficial) e2e tests are required
* Assignee @fgalan

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,8 @@ Current stable release is able to persist Orion context data in:
 * [MongoDB](https://www.mongodb.org/), the NoSQL document-oriented database.
 * [STH](https://github.com/telefonicaid/IoT-STH), a Short-Term Historic database built on top of MongoDB.
 
+You may consider to visit [Cygnus Quick Start Guide](doc/quick_start_guide.md) before going deep into the details.
```
Comment (frbattid) [inline README.md]: "to visit" or "visiting" ? I think the second one...

**Comment by fgalan** [comment]
LGTM

File: README.md (hunk 1 above)
Comment (fgalan) [inline README.md]: I think both are correct English. I would leave as it is (just for not needed to change the PR ;)

NTC

Comment (kzangeli) [inline README.md]: Both may be correct, but the second one is definitely more elegant.
However ... :-)
We have too many errors of this kind to really worry, right? ;-)

NTC

Comment (frbattid) [inline README.md]: Some day we should consume a whole sprint reviewing the grammar of all the documents at telefonicaid's repos... any candidate? ;p

**Pull request merged**